packages/bako-safe/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@fuel-connectors/bako-safe",
+  "version": "0.0.1",
+  "private": true,
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": "./src/index.ts"
+  },
+  "scripts": {
+    "build": "tsup --dts-only",
+    "build:watch": "tsup --watch --dts-only",
+    "ts:check": "tsc --noEmit",
+    "test": "vitest"
+  },
+  "dependencies": {
+    "socket.io-client": "^4.7.2",
+    "axios": "^1.5.1"
```

can we use exact versions for these?

```suggestion
    "socket.io-client": "4.7.2",
    "axios": "1.5.1"
```